app/routes/_protected._index.tsx

```diff
@@ -61,13 +60,12 @@ export default function Index() {
     useTheme();
   const { data: rates } = useQuery({
     queryKey: ['exchangeRate'],
-    queryFn: ({ signal }) =>
```

Comment:
I deleted the queryFn from here because right now we don't need to do any refetching. I feel like there should be a better way to avoid duplicating the queryFn defined in the prefetchQuery

Reply:
I would keep it. I am pretty sure refetching will be needed later. also I don't see a big problem in duplicating one function call. some amount of duplication is healthy

Reply:
Just keeping it how it is